userdocs/creating-a-codelist-from-a-csv-file.md

```diff
@@ -5,15 +5,22 @@ As well as creating a codelist from scratch, you can create one by uploading a C
 From the _My codelists_ page, click _Create a codelist_.
 
 Choose a name for the codelist, select a coding system from the dropdown,
-and choose a file to upload from your hard drive.  If you are a member of an organisation, you can also choose an owner for the codelist (your own account or an organisation account).
+and choose a file to upload from your hard drive.  If you wish to create an OPCS-4 codelist, please see the notes at the bottom of this page. If you are a member of an organisation, you can also choose an owner for the codelist (your own account or an organisation account).
 
         create-a-codelist-1.png
 
-Note that the CSV file should not have a header,
-and its first column must contain valid codes in the chosen coding system.
-(This limitation will be lifted.)
+The final codelist must be stored in CSV format to be imported 
+into [OpenCodelists](https://www.opencodelists.org). The codes must be stored in exactly one column.
+There is currently a soft requirement that the first column must contain codes in the chosen coding system, preferably named according to the CSV column names provided in the table above (this limitation will be lifted). The second column is typically a text description of the code. Before uploading a codelist CSV under your OpenSAFELY codelist profile, you should remove the header row as these will be automatically added in a standardised manner.
```

Comment:
```suggestion
### Requirements for uploading codelists to OpenCodelists

* Store the final codelist in CSV format.
* Store codes in exactly one column.
* Remove the header row. Standardised headers will automatically be added for you.
* There is currently a soft requirement that the first column must contain codes in the chosen coding system. These should preferably be named according to the CSV column names provided in the table above. (We plan to eventually remove this requirement.) The second column is typically a text description of the code.
```

I've provided a suggested re-edit here.

I can't quite understand the bolded part here out of context though:

> the first column must contain codes in the chosen coding system, **preferably named according to the CSV column names provided in the table above**

It might simply be that I don't have enough understanding, having never really used Opencodelists. It's also a bit late too :watch: